frontend/money_utils.bal

```diff
@@ -23,96 +23,95 @@ const map<string> logos = {
-    return signMatches(m) && validNanos(m.nanos);
+isolated function isValid(Money money) returns boolean {
+    return signMatches(money) && validNanos(money.nanos);
 }
 
-# checks if the sign matches
+# Checks if the sign matches
 #
-# + m - object to be validated
+# + money - object to be validated
 # + return - validity status
-isolated function signMatches(Money m) returns boolean {
-    return m.nanos == 0 || m.units == 0 || (m.nanos < 0) == (m.units < 0);
+isolated function signMatches(Money money) returns boolean {
+    return money.nanos == 0 || money.units == 0 || (money.nanos < 0) == (money.units < 0);
 }
 
-# checks if nanos are valid
+# Checks if nanos are valid
 #
 # + nanos - nano input
 # + return - validity status
 isolated function validNanos(int nanos) returns boolean {
     return -999999999 <= nanos && nanos <= +999999999;
 }
 
-# checks if the money is zero
+# Checks if the money is zero
 #
-# + m - object to be validated
+# + money - object to be validated
 # + return - zero status
-isolated function isZero(Money m) returns boolean {
-    return m.units == 0 && m.nanos == 0;
+isolated function isZero(Money money) returns boolean {
+    return money.units == 0 && money.nanos == 0;
 }
 
-# returns true if the specified money value is valid and is positive.
+# Returns true if the specified money value is valid and is positive.
 #
-# + m - object to the validated
+# + money - object to the validated
 # + return - positive status
-isolated function isPositive(Money m) returns boolean {
-    return isValid(m) && m.units > 0 || (m.units == 0 && m.nanos > 0);
+isolated function isPositive(Money money) returns boolean {
+    return isValid(money) && money.units > 0 || (money.units == 0 && money.nanos > 0);
 }
 
-# returns true if the specified money value is valid and is negative.
+# Returns true if the specified money value is valid and is negative.
 #
-# + m - object to the validated
+# + money - object to the validated
 # + return - negative status
-isolated function isNegative(Money m) returns boolean {
-    return isValid(m) && m.units < 0 || (m.units == 0 && m.nanos < 0);
+isolated function isNegative(Money money) returns boolean {
+    return isValid(money) && money.units < 0 || (money.units == 0 && money.nanos < 0);
 }
 
-# returns true if values l and r have a currency code and they are the same values.
+# Returns true if values firstValue and r have a currency code and they are the same values.
 #
-# + l - first money object
-# + r - second money object
+# + firstValue - first money object
+# + secondValue - second money object
 # + return - currency type equal status
-isolated function areSameCurrency(Money l, Money r) returns boolean {
-    return l.currency_code == r.currency_code && l.currency_code != "";
+isolated function areSameCurrency(Money firstValue, Money secondValue) returns boolean {
+    return firstValue.currency_code == secondValue.currency_code && firstValue.currency_code != "";
```

Comment:
```suggestion
    return firstValue.currency_code != "" && firstValue.currency_code == secondValue.currency_code;
```

Reply:
Fixed in https://github.com/ballerina-guides/gcp-microservices-demo/pull/30/commits/a46704a9316e3ec5a64e673fae780eda07d877a8